src/server/utils/wallets/getAwsKmsAccount.ts

```diff
@@ -0,0 +1,120 @@
+import { type KMSClientConfig } from "@aws-sdk/client-kms";
+import { KmsSigner } from "aws-kms-signer";
+import type { Hex, ThirdwebClient } from "thirdweb";
+import { keccak256, type Address as TwAddress } from "thirdweb";
```

Comment:
Nit: Why override the name here?

Reply:
remnant of a different file where it caused a conflict, but now the conflicting import is gone. Fixing.